css/main.css

```diff
@@ -0,0 +1,142 @@
+* {
+	box-sizing: border-box;
+}
+
+
+
+.response {
+	/*margin: auto;
+	text-align: center;*/    
+    margin-left: 2em;
+    position: relative;
+    top: 50%;
+    transform: translateY(-50%);
+	color: white;
+    font-size: 32px;
+    font-weight: bold;
+
+}
+
+.response > label {
+    background-color: rgba(0, 0, 0, 0.5);
+}
+
+
+.entrance {
+	background: url(../img/pirate-bar-entrance.jpg) no-repeat center center fixed;
+    background-size: cover;
+	width: 100%;
+	height: 100%;
+	position: absolute;
+	color: #fff;
+	display: none;
+}
+
+.entrance > div {
+    margin-top: 15em;
+
+}
+
+.inside {
+	background: url(../img/pirate-bar1.jpg) no-repeat center center fixed;
+    background-size: cover;
+    width: 100%;
+    height: 100%;
+    position: absolute;
+    color: #fff;
+	
+}
+
+/*.pirate-questions > h2 {
+    background-color: rgba(0, 0, 0, 0.5);
+}
+*/
+
+.enter-button {        
+    -moz-box-shadow: 3px 4px 0px 0px #1564ad;
```

Comment:
box shadow doesn't need the prefixes, unless you're aiming to support older versions of internet explorer. check caniuse.com if you're every unsure.